src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -23,7 +23,7 @@ export class SavedObjectsClient {
    * @returns {promise} - { id, type, version, attributes }
   */
   async create(type, attributes = {}, options = {}) {
-    const method = options.overwrite ? 'index' : 'create';
+    const method = get(options, 'overwrite', false) === false && options.id ? 'create' : 'index';
```

Comment:
Above it says "force id on creation, not recommended", but it looks like with these new changes, the only way to create an object is to supply options.id.  Or does `index` handle the automatic id creation if it doesn't exist?